fud2/README.md

```diff
@@ -0,0 +1,27 @@
+# `fud`: The Calyx Build Tool
+
+Reimplementation of the [fud][] compiler driver for Calyx.
+
+### Installation
+
+To install from source, run the following from `calyx/fud2`:
+```
+cargo install --path .
+```
+
+This will install the binary `fud2` to the default `cargo` binary location.
+
+### Configuration
+
+The minimal required configuration requires setting the `calyx.base` key so that `fud` knows where the Calyx compiler is. Open the configuration file by running:
+```
+fud edit-config
+```
```

Comment:
Can we (also) put this in the longer docs for fud2 that I have written, in `docs/running-calyx/fud2.md`?

Reply:
Is this page published on docs.calyxir.org?

Reply:
Hmm, I see the page but I don't see it being deployed

Reply:
#1892 was the problem